src/main/java/net/sf/jabref/logic/importer/fileformat/PdfContentImporter.java

```diff
@@ -479,6 +479,8 @@ public ParserResult importDatabase(Path filePath, Charset defaultEncoding) {
             return ParserResult.fromErrorMessage(Localization.lang("Decryption not supported."));
         } catch(IOException exception) {
             return ParserResult.fromErrorMessage(exception.getLocalizedMessage());
+        } catch (FetcherException e) {
+            return ParserResult.fromErrorMessage("Error while fetching");
```

Comment:
Why isn't the message from the FetcherException used? At least, the DOI fetcher presents a nice text, which is localized.